cwe_api/cvss_inferencer.py

```
@@ -0,0 +1,145 @@
+import logging
+import re
+import pickle
+
+DEVICE = 'cpu'
+OUTPUTS_NAME = ['attackVector', 'attackComplexity', 'privilegesRequired', 'userInteraction', 
+           'scope', 'confidentialityImpact', 'integrityImpact', 'availabilityImpact', 'baseSeverity']
+
+NUM_LABELS_PATH = 'utils/num_labels.pickle'
+MODEL_PATH = 'utils/cvss.pth'
+
+def set_global_logging_level(level=logging.ERROR, prefices=[""]):
```

Ruff

Missing return type annotation for public function `set_global_logging_level`

Add return type annotation: `None`

(ANN201)